web/src/containers/loans/Loan.js

```diff
@@ -42,21 +42,30 @@ const LoanActive = ({ loan }) => {
     <Box width="100%">
       <Box py={2}>
         <Typography variant="h3">Documents</Typography>
-        <Typography variant="subtitle1" color="textSecondary">
-          Documents uploaded by the student
-        </Typography>
+        {documents.length != 0 && (
+          <Typography variant="subtitle1" color="textSecondary">
+            Documents uploaded by the student
+          </Typography>
+        )}
+        {documents.length == 0 && (
+          <Typography variant="subtitle1" color="textSecondary">
+            The student did not upload any documents
+          </Typography>
+        )}
```

Comment:
this would be cleaner with a ternary operator. still, i think it would be better to say that there are no documents in the card below